pgcli/main.py

```diff
@@ -196,6 +196,15 @@ def run_cli(self):
                 if quit_command(document.text):
                     raise Exit
 
+                # Editor command is a special case. We don't have to execute it
+                # and print out the result. We have to extend our output and
+                # continue typing in SQL / commands.
+                if editor_command(document.text):
+                    sql = list(pgexecute.run(document.text))[0][0]
+                    document = cli.read_input(
+                        initial_document=Document(sql, cursor_position=len(sql)),
```

Comment:
`sql` is None when I don't type anything in the editor. So calling `len(sql)` causes pgcli to crash.